### DISCUSSION TITLE
[WP4629 Discussion]: cross-linked pathway is not available (not approved)

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/wikipathways-help#48] [General]
### Discussion

In the description, a pathway is mentioned https://www.wikipathways.org/index.php/Pathway:WP4628. However, this pathway is not approved at the moment, and therefore only available from the classic webpage: https://classic.wikipathways.org/index.php/Pathway:WP4628. Some of the comments from there (e.g. consider removing node fill colour) apply here as well. 

### Should this pathway be deleted?

- [ ] Pathway should be deleted

**Comment by khanspers** [comment]
I made updates to WP4629, since it only used fill color for enzymes. But for WP4628 its more complicated because of several combinations of fill/edge color. I made updates on classic and contacted the first author by email to get feedback.

**Reply by khanspers** [reply]
There is a related issue here: https://github.com/wikipathways/wikipathways-help/issues/103. Updates are planned for both pathways.
